frontend/app/routes/protected/person-case/privacy-statement.tsx

```diff
@@ -70,6 +71,21 @@ export default function PrivacyStatement({ loaderData, actionData, params }: Rou
 
   return (
     <>
+      <div className="float-right">
+        <Button id="abandon-button" endIcon={faXmark} variant="link">
+          {t('protected:person-case.abandon-button')}
+        </Button>
```

Comment:
The spacing around the text and the icon in these components is just a tiny bit too big, in my opinion.

What it is:

![image](https://github.com/user-attachments/assets/9dffac4d-b629-4413-b59d-c394755a18ca)

What I think it should be:

![image](https://github.com/user-attachments/assets/72d71e8c-51c5-416d-92ac-8efaf7920621)

Small difference but I think the second one associates the icon more closely with to the text.